src/engraving/rendering/dev/systemlayout.cpp

```diff
@@ -596,7 +596,7 @@ void SystemLayout::hideEmptyStaves(System* system, LayoutContext& ctx, bool isFi
 
     Fraction stick = system->measures().front()->tick();
     Fraction etick = system->measures().back()->endTick();
-    auto spanners = ctx.dom().spannerMap().findOverlapping(stick.ticks(), etick.ticks() - 1);
+    auto& spanners = ctx.dom().spannerMap().findOverlapping(stick.ticks(), etick.ticks() - 1);
```

Comment:
`const auto&` or `const SpannerMap::IntervalList&`

Reply:
Ah, sorry for having misled you

Reply:
Hmm, MSVC believes it to be `const` without explicitly being qualified as such